### PR TITLE
Feature/cloudsql demo

### PR DESCRIPTION
**Pull request by prabhaarya**
Cloudsql instance creation

### CODE REVIEW

**Review by hejnal — APPROVED** [review]
LGTM

File: examples/data-ingestion-cloudsql-demo/terraform/cloudsql/main.tf [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  postgres_client_cert = [
+    "terraform-test"
+  ]
+  authorized_networks = var.authorized_networks
+
+  users = {
+    # generatea password for user1
+    cloudsql-vmo2 = null
+    # assign a password to user2
+    user2 = "mypassword"
```
Comment (lcaggio) [inline examples/data-ingestion-cloudsql-demo/terraform/cloudsql/main.tf:25]: why not using the var.user_password ?

File: examples/data-ingestion-cloudsql-demo/terraform/cloudsql/variables.tf [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  default     = ""
+}
+ 
+variable "postgres_client_cert" {
+  description = "Map of cert keys connect to the application(s) using public IP."
+  type        = list(string)
+  default     = null
+}
+
+variable "network" {
```
Comment (lcaggio) [inline examples/data-ingestion-cloudsql-demo/terraform/cloudsql/variables.tf:63]: As the TF is implemented, this is not supporting shared-vpc: I would mention it or update the code to support shared-vpc

File: examples/data-ingestion-cloudsql-demo/terraform/cloudsql/output.tf [hunk 3]
```diff
@@ -0,0 +1,7 @@
+# Copyright 2022 Google. This software is provided as-is, without warranty or representation for any use or purpose.
+# Your use of it is subject to your agreement with Google.
+
```
Comment (lcaggio) [inline examples/data-ingestion-cloudsql-demo/terraform/cloudsql/output.tf:3]: I would add CloudSQL info as output: IP, region, ...

**Review by agold-rh — APPROVED** [review]

**Pull request merged**